OSDOCS-2329: Rel Notes for GCP Workload Identity (#42685)
openshift/openshift-docs: release_notes/ocp-4-10-release-notes.adoc

Users also have the option to use the `REGISTRY_AUTH_FILE` environment variable, which serves as an alternative to the existing `--registry-config` CLI flag. The `REGISTRY_AUTH_FILE` environment variable is also compatible with `podman`.

[id="ocp-4-10-auth-gcp-workload-identity"]
==== Support for Google Cloud Platform Workload Identity

> You can now use the Cloud Credential Operator (CCO) utility `ccoctl` to configure the CCO to use the Google Cloud Platform Workload Identity. When the CCO is configured to use GCP Workload Identity, it assigns IAM service accounts that provide short-term, limited-privilege security credentials to components.

Reviewer: ```suggestion
You can now use the Cloud Credential Operator (CCO) utility `ccoctl` to configure the CCO to use the Google Cloud Platform Workload Identity. When the CCO is configured to use GCP Workload Identity, the in-cluster components can impersonate IAM service accounts using short-term, limited-privilege credentials.
```
